src/schemas/v2.ts

```diff
@@ -1,6 +1,9 @@
 // This file is auto-generated; do not manually edit
```

Comment:
We need to update the tool that produces this file. https://github.com/acteng/atip/tree/ts_forms is an attempted start, but it raises some questions I haven't been able to answer yet:

- How do we want to model enums like this? The style below means it's possible for both a Route and Crossing to be defined, or for neither -- both of those feel wrong to me. The branch I linked just does "discriminated unions" to match the actual JSON currently produced: `type Intervention = { Route: Route } | { Crossing: Crossing };`. https://mariusschulz.com/blog/tagged-union-types-in-typescript is one article I found talking about these options.
- The bit where I got stuck: should all fields everywhere be optional, including enums? When we're working with finalized data, it would kind of be nice to assume everything is properly filled out. But in reality, the UI doesn't enforce that right now -- you can create an object, not fill out any of its form, and save. Or you can start clicking through the form, but not fill everything out. Maybe the reality is that every field _is_ optional and maybe missing. If so, we can adjust `generate_ts.ts` accordingly

Reply:
Maybe I'm too comment-averse because I didn't choose to ignore this comment, I had no idea it was there 😅